### DISCUSSION TITLE
Create Sionna scene using Python.

### DISCUSSION THREAD

**Discussion by manoj-kumar-joshi** [NVlabs/sionna#312] [Show and tell]
Hi,

I have created a Jupyter notebook which results in Sionna Scene from openstreetmap data by extruding building polygons and creating Road mesh. I find this useful  to create simple scenes for the area I want to select. 
Link is https://github.com/manoj-kumar-joshi/sionna_osm_scene/blob/main/OSM_to_Sionna.ipynb
<img width="634" alt="SionnaScene1" src="https://github.com/NVlabs/sionna/assets/61615138/3599302b-d18d-42a4-8830-7e24a62e24fe">

I am posting link here but if you can also post it in docs somewhere may be if you think it will be useful to others.. 

Manoj

**Comment by merlinND** [comment]
Thank you very much for sharing @manoj-kumar-joshi!

**Reply by merlinND** [reply]
@manoj-kumar-joshi, your tool will be featured in the [Made with Sionna](https://nvlabs.github.io/sionna/made_with_sionna.html) section of the documentation starting from the next release.

**Reply by manoj-kumar-joshi** [reply]
@merlinND I also need help to upload a paper at arxiv on 3D scene using Aerial LIDAR data. I have the paper along with first test [results](https://github.com/NVlabs/sionna/discussions/265).

As this will be my first upload at arxiv, I need someone to endorse me. It will be great if someone from this community can help me.

------------Endorsement email -------
Manoj Kumar Joshi requests your endorsement to submit an article to the
cs.CV section of arXiv. To tell us that you would (or would not) like to
endorse this person, please visit the following URL:

https://arxiv.org/auth/endorse?x=XLCLSD

If that URL does not work for you, please visit

http://arxiv.org/auth/endorse.php

and enter the following six-digit alphanumeric string:

Endorsement Code: XLCLSD
---------------------------------------------
